man/syncthing-config.5

```diff
@@ -206,7 +206,7 @@ may no longer correspond to the defaults.
         <releasesURL>https://upgrades.syncthing.net/meta.json</releasesURL>
         <overwriteRemoteDeviceNamesOnConnect>false</overwriteRemoteDeviceNamesOnConnect>
         <tempIndexMinBlocks>10</tempIndexMinBlocks>
-        <unackedNotificationID>authenticationUserAndPassword</unackedNotificationID>
+        <unackedNotificationID>guiAuthentication</unackedNotificationID>
```

Comment:
This change needs to be part of the docs PR, as the man pages are automatically generated and imported on a scheduled task.

Reply:
Fixed in 9811751861a3551791c39f4d642fce98ffe8f7c0.